back/spec/models/custom_field_spec.rb

```diff
@@ -15,6 +15,11 @@
       cf3 = create(:custom_field)
       expect([cf1, cf2, cf3].map(&:key).uniq).to match [cf1, cf2, cf3].map(&:key)
     end
+
```

Comment:
Would it be better to also have a test that non-latin characters are removed/replaced? Perhaps a test that tests only latin characters (or digits?) remain after processing a string containing non-latin characters?